src/app/backend/handler/apihandler.go

```diff
@@ -816,13 +850,14 @@ func (apiHandler *APIHandler) handleDeploy(request *restful.Request, response *r
 }
 
 // Handles deploy from file API call.
-func (apiHandler *APIHandler) handleDeployFromFile(request *restful.Request, response *restful.Response) {
+func (apiHandler *APIHandler) handleDeployFromFile(apiClient *clientK8s.Clientset, request *restful.Request, response *restful.Response) {
 	deploymentSpec := new(deployment.AppDeploymentFromFileSpec)
 	if err := request.ReadEntity(deploymentSpec); err != nil {
 		handleInternalError(response, err)
 		return
 	}
 
+	// TODO inject bearer token
```

Comment:
So I looked deeper into this, but TBH there's a whole jungle of apiClients in the code and it's quite hard to navigate. In the end I sttill think it would be best to modify clientConfig directly, but couldn't find a good way, yet.